feature/converter/converter.py

```diff
@@ -0,0 +1,32 @@
+class Converter:
+
+    def __init__(self, text):
+        self.text = text
+
+    def split_text(self):
+        new_text = []
```

Comment:
Try to name variables more understandable so other team members wouldn't have to deal with understanding about what's stored in that variable